packages/editor/src/components/preferences-modal/index.js

```diff
@@ -245,6 +247,16 @@ export default function EditorPreferencesModal( { extraSections = {} } ) {
 										'Show text instead of icons on buttons across the interface.'
 									) }
 								/>
+								{ !! starterPatterns?.length && (
```

Comment:
Nit: `starterPatterns` will always return an array so we probably don't need the additional `?` check.

Reply:
Good spotting. That's why I added the [default return value](https://github.com/WordPress/gutenberg/pull/65026/files#diff-c990f1de8b1af17fc68a01c104b978b55eca9cbe02a030fff2171cde211e4bdcR51) but forgot to update it. 🤦🏻